x/auth/ante/validator_tx_fee.go

```diff
@@ -19,13 +19,14 @@ func CheckTxFeeWithValidatorMinGasPrices(ctx sdk.Context, tx sdk.Tx, simulate bo
 	}
 
 	feeCoins := feeTx.GetFee()
+	feeParams := paramsKeeper.GetFeesParams(ctx)
+	feeCoins = feeCoins.NonZeroAmountsOf(append([]string{sdk.DefaultBondDenom}, feeParams.GetAllowedFeeDenoms()...))
```

Comment:
do we need a migration handler for the new feeParams?

Reply:
we don't need migration, since fees are marshal/unmarshaled via json so adding a new field won't break the compatibility. Added a test to verify this